### PR TITLE
Add HowTo for adding phase column 

### LINKED ISSUE #4020: Documentation on modifying events table
Since it is not possible to rewrite `Observation.events`, we should show some documentation on how to modify the events list. 

1. The minimal solution would be to provide a HowTo entry with maybe the following lines of code

```
import numpy as np
from gammapy.data import DataStore, Observation, EventList

datastore = DataStore.from_dir("$GAMMAPY_DATA/hess-dl3-dr1/")
obs = datastore.obs(23523)

# compute the phase separately, using, eg: PINT
phase = np.random.random(len(obs.events.table))

# attach the phase column to the events table
table = obs.events.table
table["PHASE"] = phase
events_new = EventList(table)

# create a new observation
irfs = {}
for _ in obs.available_irfs:
    irfs[_]= getattr(obs,_)
obs_new = = Observation.create(pointing=obs.pointing_radec, 
                  location=obs.observatory_earth_location,
                  obs_id=obs.obs_id,
                  tstart=obs.tstart,
                  tstop=obs.tstop,
                  irfs=irfs)
```

2. We can modify `Observation.create()` or add a `Observation.copy()` to simplify this. One of the problems with the above lines of code is that most of the information contained in `obs.fixed_pointing_info.meta` is lost in `obs_new`

I can implement either. Preference @registerrier @maxnoe @adonath ?

### PR DESCRIPTION
**Pull request by AtreyeeS**
This fixed #4020 by

- Adding an entry in the HowTo 
- Allowing `rad_max` entry in `Observation.copy()`

The two are not exactly related, but since its a rather minor edit, I put them in a single PR.

### CODE REVIEW

File: docs/user-guide/howto.rst [hunk 1]
```diff
@@ -233,3 +233,47 @@ to your code:
     pbar.SHOW_PROGRESS_BAR = True
 
 .. accordion-footer::
+
+.. accordion-header::
+    :id: collapseHowToSeventeen
+    :title: Add PHASE information to your data
+
+For doing pulser analysis, you must compute the phase associated
```
Comment (registerrier) [inline docs/user-guide/howto.rst:241]: pulsar

File: docs/user-guide/howto.rst [hunk 2]
```diff
@@ -233,3 +233,47 @@ to your code:
...
+to each event and then create a new `EventList` and a new `Observation`.
+Modifying the `EventList` of an `Observation` in-place is prohibited because of the
+underlying lazy loading implemented in reading observations.
```
Comment (registerrier) [inline docs/user-guide/howto.rst:244]: Maybe specify here that the code to compute the phase is not provided yet .

**Review by registerrier — APPROVED** [review]
Thanks @AtreyeeS .

Two minor comments inline.

**Review by registerrier — APPROVED** [review]
Thanks @AtreyeeS . This looks good!

File: docs/user-guide/howto.rst [hunk 3]
```diff
@@ -236,6 +236,53 @@ to your code:
...
+    # use the phase information - dummy in this example
+    phase = np.random.random(len(obs.events.table))
+
+    # create a new `EventList`
+    table = obs.events.table
+    table["PHASE"] = phase
+    events_new = EventList(table)
+
+    # create a new observation
+    o2 = Observation(obs_id=obs.obs_id,
```
Comment (adonath) [inline docs/user-guide/howto.rst:266]: I think this needs simplification. What about adding `Observation.copy(in_memory=True)`?

**Comment by AtreyeeS** [comment]
@adonath ! I rebased on #4083. This PR is ready from my side

**Review by registerrier — APPROVED** [review]
Thanks @AtreyeeS . This looks good!

**Review by adonath — APPROVED** [review]
Thanks @AtreyeeS, looks good to me!

**Pull request merged**